db/migrate/20220516173040_create_foods.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :foods do |t|
+      t.string :name, null: false
+      t.string :measurement_unit, null: false
+      t.decimal :price, default: 0.0
+      t.references :user, null: false, foreign_key: true
```

Comment:
[OPTIONAL] This is more of a question, in the ERD it says user_id and not user, shouldn't we keep it to the diagram?

Reply:
Yes, you are right it should be user_id. and Active Record creates user_id column as foreign key, if you give it a model name. This is a  snapshot of the foods table in Postgres:
![user_id-column-in-foods-table](https://user-images.githubusercontent.com/44057863/168775344-52fa79b9-c2b4-48f3-b284-489d7d5d9fd6.png)